app/controllers/clearance/passwords_controller.rb

```diff
@@ -56,11 +65,15 @@ def password_reset_params
     end
   end
 
-  def find_user_by_id_and_confirmation_token
-    user_param = Clearance.configuration.user_id_parameter
+  def find_password_reset_by_user_id_and_token
```

Comment:
What do you think about naming this `find_password_reset` instead? I think it might be a more override-friendly name. For instance, if you add some functionality and need to find the password reset by it as well, this method name would be misleading.